docs/src/components/MarkdownPage/MarkdownPage.js

```diff
@@ -53,16 +67,22 @@ const MarkdownPage = (props: Props) => {
     title,
   } = props;
 
+  /*
+  Currently we only want to include the page head for non-component
+  pages. Only component pages have platform metadata, so this is a
+  simple way to check for it. In the future we may wish to change this
+  logic.
+  */
+  const includePageHead = title && !platform;
```

Comment:
Could you explain why we don't want to use this for all MD pages?

Reply:
Is it because they're subpages?

Reply:
Ah, sure. It's because component pages are wrapped in the platform tab switcher, so they don't need the page head. Do you think I should update the comment to make this clear?

Prior to this PR we were still doing this but the logic was just `title` and didn't involve platform. The reason I changed it in this PR was because I added titles to the Android/iOS/RN text MDX pages that could be used in the metadata.

Reply:
Don't we have a prop called `wrapped` or something which would be much more self-documenting?

Reply:
I thought about doing this, but I think that's actually a worse long-term experience. We'd be tying in presentational logic in the MDX files, because we'd need to do this:

```yaml
---
title: Text
platform: ios
wrapped: true
---
```

The MDX files shouldn't have to be aware of how they're presented, they should only contain stuff related to the content. 

Let's say in future we decide to stop doing this page wrapping and present each component as its own page. With a `wrapped` prop we'd need to change every mdx file to remove it, whereas with my solution we would just change the logic we use in one place. So this feels like a better solution to me.

Reply:
I was thinking about these pages. I guess it would be nice to avoid having to put `wrapped` on each one, but I also don't like inferring that a page is wrapped because the platform is specified 🤔

<img width="621" alt="Untitled" src="https://user-images.githubusercontent.com/30267516/101805143-53687b00-3b0a-11eb-8216-8434a0051805.png">

Reply:
I guess in this case though, `platform` is effectively `subpageName`. So if it has a `subpageName` then we can assume it doesn't need its own meta...

Reply:
Ah, that wrapped prop is actually unused, it's a holdover from before. It has no effect